controls/genericPlaybook.yaml

```diff
@@ -7,6 +7,7 @@
   tasks:
     - set_fact:
         stereum: "{{ stereum_static | combine(stereum_args, recursive=True) }}"
+      when: stereum_args is defined
```

Comment:
shouldn't there be a
```
- set_fact:
    stereum: "{{ stereum_static}}"
  when: stereum_args is undefined
```
otherwise var `stereum` would be empty if `stereum_args` isn't set

Reply:
`is defined` is fine because it should only be combined if there is something to combine, but i'll change it so `stereum` is set to `stereum_static` if `stereum_args` is undefined to prevent any errors with `stereum` being undefined